src/data/store/bulk.js

```diff
@@ -1,76 +1,59 @@
 import Utils from '../../utils'
 import Urls from '../../urls'
 import Request from '../../request'
+import Async from '../../request/async'
 
-export function bulkCreate(objectsArray, asyncHandler) {
+export function bulkCreate(objects, asyncHandler) {
   const MSG_ERROR = (
-    'Invalid value for the "objectsArray" argument. ' +
-    'The argument must contain only array of objects.'
+    'Invalid bulkCreate argument. ' +
+    'The first argument must contain only array of objects.'
   )
 
-  if (!Utils.isArray(objectsArray)) {
+  if (!Utils.isArray(objects)) {
     throw new Error(MSG_ERROR)
   }
 
-  objectsArray.forEach(obj => {
-    if (!Utils.isObject(obj)) {
+  objects.forEach(obj => {
+    if (!Utils.isObject(obj) || Array.isArray(obj)) {
       throw new Error(MSG_ERROR)
     }
   })
 
   return Request.post({
     url         : Urls.dataBulkTable(this.className),
-    data        : objectsArray,
+    data        : objects,
     isAsync     : !!asyncHandler,
     asyncHandler: asyncHandler
   })
 }
 
-export function bulkUpdate(templateObject, where, asyncHandler) {
-  if (!templateObject || !Utils.isObject(templateObject)) {
-    throw new Error('Invalid templateObject argument. The first argument must contain object')
+export function bulkUpdate(where, changes, asyncHandler) {
```

Comment:
this is breaking change. please revert it

Reply:
the Bulk Operation methods have not been released/documented yet